chat-plugins/roomsettings.js

```diff
@@ -467,4 +476,35 @@ exports.commands = {
 		"/banword delete [words] - Removes the comma-separated list of phrases from the banword list. Requires: # & ~",
 		"/banword list - Shows the list of banned words in the current room. Requires: % @ * # & ~",
 	],
+	robot9000: 'r9k',
+	r9k: function (target, room, user) {
+		if (!target) {
+			const r9k = (room.r9k ? "enabled" : "disabled");
+			return this.sendReply(`Robot9000 mode is currently ${r9k}.`);
+		}
+		if (!this.canTalk()) return;
+		if (!this.can('editroom', null, room)) return false;
+
+		if (target === 'enable' || target === 'on' || target === 'true') {
+			if (room.r9k) return this.errorReply(`Robot9000 mode is already enabled for this room`);
+			if (!room.r9kPermanentCache) Chat.r9kLoadPermanent(room);
+			room.r9k = true;
```

Comment:
It should return something after, saying that it was enabled here (if it was enabled).  Silent replies will make the user think their lagging or that it otherwise didn't work right the first time.